### DISCUSSION TITLE
Sunbird-ED local Installation

### DISCUSSION THREAD

**Discussion by Kovendan03** [Sunbird-Ed/Community#188] [Installation] [answered]
Hi Team,

I am trying to install, Sunbird ED as a local installation, followed the documentation - https://github.com/Sunbird-Ed/SunbirdEd-portal

when i tried start to start the backend services , it got failed and where the error was referring to azure storage/access keys. Can we try installing this component as a local storage service?

**Comment by gandham-santhosh** [comment, marked as answer]
@sharathkashyap @rajeevsathish can you please help here?

**Reply by rajeshkumaravel** [reply]
@Kovendan03 
Currently SunbirdEd supports only Azure Cloud Storage until release-5.0.0. Please configure Azure for services to run.
Reference environment variables
```
sunbird_azure_resourceBundle_container_name: env.sunbird_azure_resourceBundle_container_name || 'label',
sunbird_azure_report_container_name: 'reports',
sunbird_azure_account_name: env.sunbird_azure_account_name || 'sunbirded',
sunbird_azure_account_key: env.sunbird_azure_account_key || ''
```
CC: @sharathkashyap @rajeevsathish
